### DISCUSSION TITLE
How to update Hypar CLI?

### DISCUSSION THREAD

**Discussion by luzcurletto** [cake-build/cake#3293] [Q&A]
i have already installed de Hypar CLI 0.7.3 and i need to change it for a newer one, i downloaded de 0.8.8 version but i dont know how to unistallest the first one. Can u help me?

**Comment by patriksvensson** [comment]
What is Hypar CLI? How did you install it?

**Comment by luzcurletto** [comment]
I want to try the workflow between Revit and Hypar, and I follow this
instruction https://hypar-io.github.io/Elements/Revit.html
Install the latest Hypar CLI
<https://hypar-io.github.io/Elements/C-Sharp.html#installing-and-using-the-hypar-cli>.
You'll need this to use the hub command.

dotnet tool install -g hypar.cli

and when i run it on cmd.exe, it says that " it's already installed" and
when later i write "sign in" it says: "your version is not compatible with
Hypar anymore. Please update to at least 0.8.3... by visiting ".
https://www.nuget.org/packages/Hypar.CLI/" So i downloaded the 0.8.8
version but i don't know how to install it.

**Comment by devlead** [comment]
Not familiar with the tool but if it's this package https://www.nuget.org/packages/Hypar.CLI/

Then if you click the Cake tab on NuGet.org you'll see how to install and version the tool locally within your Cake script.
![image](https://user-images.githubusercontent.com/1647294/112642873-05bb1880-8e44-11eb-8f91-ee76440db86a.png)

I.e.

### build.cake

```cake
#tool dotnet:?package=Hypar.CLI&version=0.8.8

var pathToHypar = Context.Tools.Resolve("hypar") ?? Context.Tools.Resolve("hypar.exe");

StartProcess(pathToHypar, "--version");
```

### Execute

```bash
dotnet cake build.cake
```

### Result

```
0.8.8
```
